.travis.yml

```diff
@@ -14,15 +14,33 @@ cache:
   directories:
     - $HOME/.cache
 
-env:
-  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.13
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
-  - EMBER_TRY_SCENARIO=ember-default
+jobs:
+  fail_fast: true
+  include:
+    - stage: 'Tests'
+      env: EMBER_TRY_SCENARIO=ember-1.13
+    - env: EMBER_TRY_SCENARIO=ember-lts-2.4
+    - env: EMBER_TRY_SCENARIO=ember-lts-2.8
+    - env: EMBER_TRY_SCENARIO=ember-3.0
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.8
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.12
+    - env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-default
+
+    - stage: 'Deploy'
+      if: tag IS present
+      script: skip     # usually you do not want to rerun any tests
+      deploy:
+        provider: npm
+        email: info@simplabs.com
+        api_key:
+          secure: PA0X+4BPH2ACTKsOJQ5/018XKB6OZ3jUR5le/8ILCIfsCJUTSe1qKjv2qu10fhRDsUd1/HXSiqDWk4xnbencrkK5QhYkpR+6r2ryu01yGXqy3sKHxSZRdy7to0XHMUY55l5avvZPCUOohQljdEqb2zksSP26AfIDq3hXmqjvLaKyN/rV7KzG5X+a77WO0BMXefogBGA5HV8o7irKNMaU04z5XoKvzqJoVH9Mjc58YPLtC/8BYIO0dpkBiAxdEkz1pioha1SRgtQEhvfcK+8xoo+iCMglx6FvbuHI4GRHLgBGYfBDzuebqCgKgY1oiMLVoZpzhFs/qLpUGsZMZLkGQD0OPDidrnDPHaGERJIu/GC5lPuyRAtIYSfTJCT93Y30ZyHSJwgDEu4x4wO3Xy+3uXqo1y1kx8DE2moDFhrxeDlTZTQdt1+HDmwTFwLn9iZHngJLU5Yzf+5dGV3eREig0pjyBSBKK1a2jZO/YOxUYeYUkDxRjon6DyGMwi+DP59HHsr23Shntt70lUrmyBJTfBo1OQDB0OzeyLN7n+Q/ybpFqO58uVRE2w+eGepm9pDWeb1uWX+tSvBc7PTBdE+8aAQkUDfCEIZ916Oq4ijpkW/Bt83LncxmLb9Sd8/UKyG0ozkZgvgTqbJaRQef9IewYv9tfsneDqYsSdDHSwUfQDw=
+        on:
+          tags: true
```

Comment:
What's the difference between `if: tag IS present` above and 
```        
on:
  tags: true
```
?

Reply:
Good catch - I had a quick look at the docs and it seems `on:` is specifically for deployments while `if:` is more generic. I'll remove the `if:` as the `on:` seems more expressive anyways.

Reply:
done

Reply:
It seems `if:` is actually better as otherwise the deployment step shows up although it won't do anything, e.g. https://travis-ci.org/simplabs/ember-cookies/builds/604390063?utm_source=github_status&utm_medium=notification - changed back